test/functional/shada/helpers.lua

```diff
@@ -37,8 +37,6 @@ local function add_argv(...)
 end
 
 local clear = function()
-  os.execute('cp ' .. tmpname .. ' /tmp/test.shada')
-  os.remove(tmpname)
```

Comment:
@ZyX-I Also removing this, LMK if not.

Reply:
The whole reason of `cp` is because `os.remove` *must* be there for cleanup.

Reply:
@ZyX-I Thanks. I wasn't sure because `tmpname` is a module-level variable. But now I see that it's re-created by `nvim -i ...`.